userspace/libsinsp/metrics_collector.cpp

```diff
@@ -783,6 +770,19 @@ void libs_metrics_collector::snapshot()
 			}
 		}
 	}
+
+	/*
+	* plugins metrics
+	*/
+
+	if(m_metrics_flags & METRICS_V2_PLUGINS)
```

Comment:
Why was this moved?

Reply:
Just curious :)

Reply:
If we insert plugin metrics first in the vector, they then get overwritten by scap/sinsp metrics.
I didn't look closely at it, but the cause may be: https://github.com/falcosecurity/libs/blob/master/userspace/libsinsp/metrics_collector.cpp#L487